roles/receptor/tasks/setup-RedHat.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Install receptor packages
```

Comment:
After licensing a rhel node with an aap sku, we would need to enable the proper aap channel to install from before being able to run dnf. Is that part of the requirements for the customer? Or should we enable that here?

Reply:
Do you mean to tell me that the receptor package is not freely available?

Reply:
Correct, currently, to get access to the receptor rpm you need an AAP subscription, and need to attach that pool with subscription-manager on your RHEL box.

Reply:
The installer assumption is that the nodes are already subscribed to aap. Then before proceeding to install we run `subscription-manager repos --add=repo-name` 

Some caveats here:
- `ansible_os_family` is a bigger set than RHEL, it includes fedora or OEL, or Amazon Linux
- `repo-name` depends in aap X.Y and rhel X versions. kind of `ansible-automation-platform-2.3-for-rhel-8-rpms`

Reply:
> After licensing a rhel node with an aap sku, we would need to enable the proper aap channel repo using subscription-manager to install from before being able to run dnf. Is that part of the requirements for the customer? Or should we enable that here?

I do not believe we should enable that here.  This collection is supposed to be open to the community.